qemu/tests/cfg/cpu_topology_test.cfg

```diff
@@ -13,3 +13,15 @@
             required_qemu = [6.2.0,)
             # Check PPTT table mapping by looking at core_id
             check_core_id_cmd = 'cat /sys/devices/system/cpu/cpu%s/topology/core_id'
+        s390x:
+            no RHEL.6 RHEL.7
+            no Host_RHEL.m6 Host_RHEL.m7 Host_RHEL.m8
+            required_qemu = [8.2.0,)
+    variants:
+        - check_siblings_core_id_cmd:
```

Comment:
There is only 1 variant, so other architectures cannot find the test case, as the only one variant is defined with `only s390x`.

Reply:
updated

Reply:
I'll add other variant later but with other patch